src/components/Sum/Sum.jsx

```diff
@@ -1 +1,10 @@
-// export const Sum = () => ();
+export const Sum = ({
+  a = 0,
+  b = 0,
+}) => (
+  <>
+    <p>
+      {`Sum of ${a} and ${b} is ${a + b}`}
+    </p>
+  </>
```

Comment:
```suggestion

```

Reply:
You are using just one element, so there is no need to add fragments here

Reply:
Clear. Thanks.